cli/azd/pkg/tools/docker/docker_test.go

```diff
@@ -115,21 +129,27 @@ func Test_DockerBuildEmptyPlatform(t *testing.T) {
 	docker := NewDocker(mockContext.CommandRunner)
 
 	mockContext.CommandRunner.When(func(args exec.RunArgs, command string) bool {
-		return strings.Contains(command, "docker build")
+		return strings.Contains(command, "docker build") && !strings.Contains(command, "-q")
 	}).RespondFn(func(args exec.RunArgs) (exec.RunResult, error) {
 		ran = true
 
+		// extract img id file arg
+		argsNoFile, _, value := mocks.RemoveArg("--iidfile", args.Args)
+
 		require.Equal(t, "docker", args.Cmd)
 		require.Equal(t, cwd, args.Cwd)
 		require.Equal(t, []string{
 			"build",
-			"-q",
 			"-f", dockerFile,
 			"--platform", platform,
 			"-t", imageName,
 			"--build-arg", buildArgs[0],
 			dockerContext,
-		}, args.Args)
+		}, argsNoFile)
+
+		// create the file as expected
+		err := os.WriteFile(value, []byte("Docker build output"), 0600)
```

Comment:
NIT: The phrase "Docker build output" here might be a little confusing, since it may make one think of the output that `docker build` wrote to the screen (and we'd show in our progress log).  Perhaps this should just some something like: "fake-docker-image-id"?